### PR TITLE
OGR Source

### PR DESCRIPTION
**Pull request by SoerenHoffstedt**
Added OGR Source Operator and renamed old operator to OGR Raw Source Operator.
Added source listing of OGR datasets in user service. The functionality of that is in ogr_source_datasets.
Test queries and data are included.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -92,7 +92,9 @@ add_library(mapping_core_base_lib
         util/parameters.cpp
         util/parameters.h
         datatypes/raster/raster.cpp
-        raster/opencl.cpp)
+        raster/opencl.cpp
+        util/ogr_source_datasets.cpp
+        util/ogr_source_datasets.h)
```
Comment (michaelmattig) [inline src/CMakeLists.txt]: redundant include?

File: test/systemtests/data/ogr_files/stationsWithCoordinates.csv [hunk 2]
```diff
@@ -0,0 +1,339 @@
+X,Y,Station_ID,type
```
Comment (michaelmattig) [inline test/systemtests/data/ogr_files/stationsWithCoordinates.csv]: where does this file come from? License?

**Review by michaelmattig — COMMENTED** [review]
> Configuration: 'ogrsource.files.path' not found in subtable.
has to be set up in mapping_test.conf

File: test/systemtests/data/ogr_files/stationsWithCoordinates.csv (hunk 2 above)
Comment (SoerenHoffstedt) [reply test/systemtests/data/ogr_files/stationsWithCoordinates.csv]: I got the files (including the allStations files) from Johannes. @jdroenner do you know the license?

**Comment by SoerenHoffstedt** [comment]
I added the variable to `mapping_test.conf` and updated the file to be in TOML structure. I also adjusted the paths of other variables to fit the location of the executable.
I also found a bug in Configuration not loading configuration files set in an environment variable.

File: test/systemtests/data/ogr_files/stationsWithCoordinates.csv (hunk 2 above)
Comment (jdroenner) [reply test/systemtests/data/ogr_files/stationsWithCoordinates.csv]: I guess we should not use non public data for tests... Maybe we should use something else.

http://www.naturalearthdata.com/downloads/10m-cultural-vectors/ <-- Natural Earth is free: http://www.naturalearthdata.com/about/terms-of-use/

Comment (SoerenHoffstedt) [reply test/systemtests/data/ogr_files/stationsWithCoordinates.csv]: I removed the files and pushed new tests with files from Natural Earth.

**Pull request merged**